_dev/src/components/product-feed/settings/attribute-mapping/attribute-field.vue

```diff
@@ -142,6 +148,12 @@ export default {
         ? 'attribute-is-mapped' : null;
     },
   },
+  mounted() {
+    if (this.$store.state.productFeed.attributeToEdit === null) {
+      return;
+    }
+    window.scrollTo(0, 500);
```

Comment:
From my understanding, we will trigger this scroll as many time as we have attribute field displayed on the page?
What about using anchors like in https://stackoverflow.com/a/37425373? You wouldn't need to add logic here, we would also impact that run the product feed funnel for the first time